ddtrace/internal/packages.py

```diff
@@ -0,0 +1,32 @@
+import typing
+
+
+Distribution = typing.NamedTuple("Distribution", [("name", str), ("version", str)])
+
+_DISTRIBUTIONS = None
+
+
+def get_distributions():
+    # type: () -> typing.List[Distribution]
+    global _DISTRIBUTIONS
+    if _DISTRIBUTIONS is not None:
+        return _DISTRIBUTIONS
+
+    try:
+        import importlib.metadata as importlib_metadata
+    except ImportError:
+        import importlib_metadata
+
+    pkgs = []
+    pkg_names = set()
+    for dist in importlib_metadata.distributions():
+        metadata = dist.metadata
+        name = metadata["version"]
```

Comment:
This doesn't look right 🤔

Reply:
this should have been picked up in tests